app/web/features/communities/events/EventTimeChanger.tsx

```diff
@@ -124,9 +92,12 @@ export default function EventTimeChanger({
           }}
           testId="startDate"
         />
-        <TextField
-          id="startTime"
-          {...register("startTime", {
+
+        <Controller
+          control={control}
+          defaultValue={eventStartTime || ""}
+          name="startTime"
+          rules={{
```

Comment:
Okay taking a closer look at this now!

So I see here you have wrapped this in a controller and are also passing `control` and wrapping the Timepicker in a Controller again in the Timepicker component.

You only need one, otherwse you're doubling it which might cause some issues.

I'd recommend removing this Controller and just pass Timepicker, since you already have a Controller within Timepicker. You can pass the rules to the Timepicker component in `EventTimeChanger` instead.